### PR TITLE
Olivia/GAM main page routing and web res styling

### PR DESCRIPTION
**Pull request by oliviasztanga**
Sorry, guys! Still learning how to work on a team. Here are some screen caps:

Desktop:
GAM Home Web
GAM Home Web - Listings

Mobile:
GAM Home Mobile
GAM Home Mobile Listings

Need some feedback on the following still:
1. Desktop listings break at 900px. Should we use the mobile styling at that point? What do you all think?
GAM Home Web - Listings breaking @ 900px

2. Is it okay to overwrite the /gift-a-meal route currently used for the GoFundMe?

### CODE REVIEW

File: src/components/MerchantsPage/gam/CampaignListItem.tsx [hunk 1]
```diff
@@ -10,7 +10,8 @@ import apexLogo from '../images/apex-logo.png';
 import melonpannaLogo from '../images/melonpanna-logo.png';
 
 // In the final implementation, campaign will be object declared in types.ts
-const CampaignListItem = (campaign: String) => {
+// const CampaignListItem = (campaign: String) => {
```
Comment (danthaman44) [inline src/components/MerchantsPage/gam/CampaignListItem.tsx:13]: dont forget to delete this before checking in :)

File: src/components/MerchantsPage/gam/GiftAMealPage.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import gam_1 from '../images/gam_1.svg'
+import gam_2 from '../images/gam_2.svg'
+import gam_3 from '../images/gam_3.svg'
+import gam_4 from '../images/gam_4.svg'
+
+interface Props {
+    menuOpen: boolean;
+}
+
+const GiftAMealPage = (props: Props) => {
```
Comment (danthaman44) [inline src/components/MerchantsPage/gam/GiftAMealPage.tsx:16]: Did you run `yarn format`? This file seems a bit more indented than the rest of our files

File: src/components/App/App.tsx [hunk 3]
```diff
@@ -88,6 +92,7 @@ const App = () => {
               window.location.href = 'https://www.gofundme.com/f/gift-a-meal';
               return null;
           }}/>
+          <Route path="/gift-a-meal-page">{returnComponent('giftameal')}</Route>
```
Comment (danthaman44) [inline src/components/App/App.tsx:95]: thoughts on naming the route to be just "gift-a-meal"?

File: src/components/MerchantsPage/gam/GiftAMealPage.tsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styles from './styles.module.scss';
+import CampaignListItem from './CampaignListItem'
+
+import gam_hero from '../images/gam_hero.png'
+import gam_1 from '../images/gam_1.svg'
```
Comment (jacoblb64) [inline src/components/MerchantsPage/gam/GiftAMealPage.tsx:7]: nit: maybe we can give these more descriptive names?

File: src/components/App/App.tsx (hunk 3 above)
Comment (oliviasztanga) [reply src/components/App/App.tsx:95]: Sorry! Somehow missed all of these comments this weekend. Yes, agree this should just be gift-a-meal. That route is currently being used to link to the external GoFundMe page so I didn't want to overwrite it yet.

File: src/components/MerchantsPage/gam/GiftAMealPage.tsx (hunk 4 above)
Comment (oliviasztanga) [reply src/components/MerchantsPage/gam/GiftAMealPage.tsx:7]: I agree! I'll rename these.

File: src/components/MerchantsPage/gam/GiftAMealPage.tsx (hunk 2 above)
Comment (oliviasztanga) [reply src/components/MerchantsPage/gam/GiftAMealPage.tsx:16]: Oops! I did not!

File: src/components/App/App.tsx (hunk 3 above)
Comment (danthaman44) [reply src/components/App/App.tsx:98]: Oh good catch! I didn't know about the GoFundMe route. I'll ask in the channel about the best way to disambiguate the two.

Comment (danthaman44) [reply src/components/App/App.tsx:98]: Ok based on the Slack convo, let's just use "/gift-a-meal-page" for now until the full page is ready :)

**Pull request merged**